nion/swift/ExportDialog.py

```diff
@@ -221,56 +223,163 @@ def cancel(self) -> bool:
         return True
 
 
-class ExportSVGHandler:
-    def __init__(self, display_size: Geometry.IntSize) -> None:
-        self.width_model = Model.PropertyModel(display_size.width)
-        self.height_model = Model.PropertyModel(display_size.height)
+class UnitType(enum.Enum):
+    INCHES = 0
+    CENTIMETERS = 1
+    PIXELS = 2
 
-        self.int_converter = Converter.IntegerToStringConverter()
+class ExportSVGHandler:
 
+    def __init__(self, display_size: Geometry.IntSize) -> None:
+        self.display_size = display_size
+        self.aspect_ratio = display_size.width / display_size.height
+        self.initial_width = str(display_size.width)
+        self.initial_height = str(display_size.height)
+        self.unit_model = UnitType.PIXELS  # Default to pixels
+        self.float_converter = Converter.FloatToStringConverter()
+        self.previous_unit_index = UnitType.PIXELS
+        self.width_value_line_edit: UserInterface.LineEditWidget
+        self.height_value_line_edit: UserInterface.LineEditWidget
+        self.size = Geometry.IntSize(0, 0)  # Changed from FloatSize to IntSize
         u = Declarative.DeclarativeUI()
-        width_row = u.create_row(u.create_label(text=_("Width (in)"), width=80), u.create_line_edit(text="@binding(width_model.value, converter=int_converter)"), spacing=12)
-        height_row = u.create_row(u.create_label(text=_("Height (in)"), width=80), u.create_line_edit(text="@binding(height_model.value, converter=int_converter)"), spacing=12)
-        main_page = u.create_column(width_row, height_row, spacing=12, margin=12)
-
+        self.width_line_edit = u.create_line_edit(
```

Comment:
`self.width_line_edit`, `self.height_line_edit`, and `self.units_combo_box` are only used in this function and so should be local variables.

However. I'd really like to not use intermediate variables when not necessary as it makes the structure of the dialog more difficult to understand. Please consider doing something like here:

https://github.com/nion-software/nionswift/blob/b21329ff63f00027ba74e43fe0ee693b6de79e74/nion/swift/Inspector.py#L2347-L2368